tests/functional/test_install_user.py

```diff
@@ -68,24 +68,26 @@ def test_install_subversion_usersite_editable_with_distribute(
         )
         result.assert_installed('INITools', use_user_site=True)
 
+    @pytest.mark.network
     def test_install_curdir_usersite(self, script, virtualenv, data):
         """
         Test installing current directory ('.') into usersite
         """
         virtualenv.system_site_packages = True
+        script.pip("install", "wheel")
```

Comment:
Are you sure this is actually needed?

Reply:
I have not validated this myself but from experience with related tests, I would strongly suspect that it is.